### DISCUSSION TITLE
IOS有哪个App支持XTLS Vision协议吗？

### DISCUSSION THREAD

**Discussion by Jinnrry** [XTLS/Xray-core#1573] [General]
找了好多个，都是v2ray内核，不支持XTLS Vision协议

**Comment by FranzKafkaYu** [comment]
暂时没有，各客户端对Vision的支持都比较滞后，IOS端更是如此。

**Comment by simplerick-simplefun** [comment]
我也有这个需求。一个都没有吗？

**Comment by yoohooyoo** [comment]
stash一般跟进很快，希望能支持。 @StashNetworks

**Comment by yiyievil** [comment]
小火箭没有选项，但是直接扫节点二维码可以添加并且翻墙

**Reply by Jinnrry** [reply]
我测试并不行。你确定你的二维码是vision协议？

**Reply by yiyievil** [reply]
很确定啊

**Reply by mostend** [reply]
> 我测试并不行。你确定你的二维码是vision协议？
因为他的配置文件里面写了流控xtls-rprx-vision,none

**Comment by awaking202002** [comment]
最新版的小火箭 2.2.25 ，今天更新了，支持 vision 了~ 在 XTLS 项里，默认是 none，记得修改。

**Reply by chika0801** [reply]
注意事项

https://github.com/XTLS/Xray-core/issues/1795#issuecomment-1468148685

配置示例

https://github.com/chika0801/Xray-install/blob/main/README.md#shadowrocket-%E9%85%8D%E7%BD%AE%E7%A4%BA%E4%BE%8B

**Comment by txdywy** [comment]
pharos pro用的clash.meta
